### PR TITLE
Update provider for AVD and add scaling plans

### PR DESCRIPTION
**Pull request by WillyMoselhy**

### CODE REVIEW

**Comment by cmcclister** [comment]
Learn Build status updates of commit _[5215c89](https://github.com/WillyMoselhy/cloud-adoption-framework/commits/5215c892c8ac6bd2acd6f256d16df2d1ed03c9b3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/ready/azure-best-practices/resource-abbreviations.md](https://github.com/WillyMoselhy/cloud-adoption-framework/blob/master/docs/ready/azure-best-practices/resource-abbreviations.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/0286ad65-6ec8-0c70-0cae-2edc1504c701/202211131116312919-1000/BuildReport?accessString=12e0f181c69b7c5782144ab606519f21ab273eee2c53ef66e04c0d1693c6f4d8).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0286ad65-6ec8-0c70-0cae-2edc1504c701?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by WillyMoselhy** [comment]
I am using the sameabbreviation for scaling plans as we see in the AVD Accelerator

File: docs/ready/azure-best-practices/resource-abbreviations.md [hunk 1]
```diff
@@ -197,9 +197,10 @@ The Azure Naming Tool generates Azure-compliant names and can help you standardi
 
 | Resource | Resource provider namespace | Abbreviation |
 |--|--|--|
-| Virtual desktop host pool | | `vdpool` |
-| Virtual desktop application group | | `vdag` |
-| Virtual desktop workspace | | `vdws` |
+| Virtual desktop host pool | Microsoft.DesktopVirtualization/hostPools | `vdpool` |
+| Virtual desktop application group | Microsoft.DesktopVirtualization/applicationGroups | `vdag` |
+| Virtual desktop workspace | Microsoft.DesktopVirtualization/workspaces | `vdws` |
+| Virtual desktop scaling plan | Microsoft.DesktopVirtualization/scalingPlans | `vdscaling`|
```
Comment (stephen-sumner) [inline docs/ready/azure-best-practices/resource-abbreviations.md:203]:
```suggestion
| Virtual desktop host pool | `Microsoft.DesktopVirtualization/hostPools` | `vdpool` |
| Virtual desktop application group | `Microsoft.DesktopVirtualization/applicationGroups` | `vdag` |
| Virtual desktop workspace | `Microsoft.DesktopVirtualization/workspaces` | `vdws` |
| Virtual desktop scaling plan | `Microsoft.DesktopVirtualization/scalingPlans` | `vdscaling`|
```

**Review by stephen-sumner — APPROVED** [review]
Great updates. Thanks for adding the namespace!

**Comment by stephen-sumner** [comment]
#sign-off

**Pull request merged**